content/en/docs/concepts/security/index.md

```diff
@@ -108,12 +108,12 @@ platforms:
    service account refers to the existing service account just like the
    identities that the customer's Identity Directory manages.
 
-## Public Key Infrastructure (PKI) {#pki}
+## Identity Issuance {#pki}
```

Comment:
```suggestion
## Identity and Certificate Management {#pki}
```

Reply:
Perhaps we should also change this "pki" reference?  Maybe "identity"?  If so, we could also remove the lower-case "pki" definition from the ".spelling" file that was recently added.

Reply:
Make "certificate management" lowercase. (Unless it's a product name or a trademarked term or something like that.) (See https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings_1)

Reply:
```suggestion
## Identity and certificate management {#pki}
```